apps/dev-portal-web/domains/docs/components/AmountDistributionCalculator.tsx

```diff
@@ -0,0 +1,357 @@
+import { faker } from '@faker-js/faker'
```

Comment:
Unknown dependency. Do we really need it?

Reply:
Removed. Added the function to generate numbers instead.